Core/HLE/sceUtility.cpp

```diff
@@ -742,6 +748,28 @@ static int sceUtilityGameSharingGetStatus()
 	return 0;
 }
 
+static u32 sceUtilityLoadUsbModule(u32 module)
+{
+	if (module < 1 || module > 5)
+	{
+		ERROR_LOG(SCEUTILITY, "sceUtilityLoadUsbModule(%i): invalid module id", module);
+	}
+
+	ERROR_LOG(SCEUTILITY, "UNIMPL sceUtilityLoadUsbModule(%i)", module);
+	return 0;
+}
+
+static u32 sceUtilityUnloadUsbModule(u32 module)
+{
+	if (module < 1 || module > 5)
+	{
+		ERROR_LOG(SCEUTILITY, "sceUtilityUnloadUsbModule(%i): invalid module id", module);
+	}
+
+	ERROR_LOG(SCEUTILITY, "UNIMPL sceUtilityUnloadUsbModule(%i)", module);
```

Comment:
Might be good to make these _REPORT so we can more easily see what games are trying to load them, in case some still have issues with these stub implementations.

-[Unknown]